solidity/apps/test/governance/governanceRouter.test.ts

```diff
@@ -57,54 +52,6 @@ describe('GovernanceRouter', async () => {
     ).to.be.revertedWith('Initializable: contract is already initialized');
   });
 
-  it('accepts message from enrolled inbox and router', async () => {
```

Comment:
So I'm late to the party here but should we really remove these tests from applications? IMO this is a pretty critical piece to an applications security and would imagine most developers should test for this in their Routers, even if they are not implementing the logic?

Reply:
The tests make more sense in `Router`, IMO

Reply:
agree that they should not need to test these on their routers
don't we want to design a secure pattern with minimal boilerplate?
alternatively we could add tests to the hardhat or app packages that are run against their implementations automatically or with some config

Reply:
Yeah, obviously they should also live in `Router`, but it seems to me that this is a very critical part of the Router pattern that should be ensured? I guess it is kind of hard to "accidentally override" the `handle` function, but its really more about setting the expectation of what should ultimately be tested. It reminds me that in let's say a traditional SaaS application, even if authentication is mostly handled by a library and their tests, it seems important to still have it be tested at the application layer.

I think concision (if that's the word) obviously is very beneficial, especially in the smart contract code, but in testing, I think it's worth to be verbose/expansive IMO